docs/docs/how-to/plugins-and-themes/creating-a-source-plugin.md

```diff
@@ -806,6 +806,107 @@ One challenge when developing locally is that a developer might make modificatio
 
 Some data sources keep event logs and are able to return a list of objects modified since a given time. If you're building a source plugin, you can store the last time you fetched data using the [cache](/docs/creating-a-source-plugin/#caching-data-between-runs) and then only sync down nodes that have been modified since that time. [`gatsby-source-contentful`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-contentful) is an example of a source plugin that does this.
 
+### Enabling Content Sync
+
+If you would like to add Content Sync to your source plugin here but aren't sure what it is [learn more about Content Sync here](https://www.gatsbyjs.com/docs/conceptual/content-sync/). To enable this feature in your source plugin you will need to make sure that your data source (or CMS) also works with Content Sync. The first thing you'll want to do is identify which nodes you'll want to create a node manifest for. These will typically be nodes that you can preview, entry nodes, top level nodes, etc. An example of this could be a blog post or an article, any node that can be the "owner" of a page. Once you've identified that you'll want to use the [unstable_createNodeManifest action](https://www.gatsbyjs.com/docs/reference/config-files/actions/#unstable_createNodeManifest) to create these manifests. A good place to call this action is before you call `createNode`.
```

Comment:
We'll probably also want a guide on how to implement this on the CMS side. I don't believe we have this yet

Reply:
Doc links need to be relative

Reply:
Yeah, if we're mentioning how to implement it in source plugins we need the CMS side as well for it to make sense. This is awesome though :)